cores-and-flashing.tex

```diff
@@ -35,7 +35,36 @@ \section{What are cores, and why does it matter?}
 of these systems (once the cores are available), and back again.
 
 The remainder of this chapter describes how to select which core to run on the MEGA65, and
-how to store a core into one of the three (or for MEGA65R2, seven) slots in the flash memory storage.
+how to store a core into one of the seven slots in the flash memory storage.
+
+\subsection{Model types}
+
+The retail model you have purchased is referred to as the MEGA65R3b (revision 3b). Throughout the course of development of the MEGA65, there have been several other model variants used by developers, each with differing specifications and available core slots, so they will be listed here, just to raise awareness of them.
```

Comment:
3A or 3B? note the capital also

Reply:
Yeah, this 3a/3b idea was just an initial stab in the dark until Paul and Deft filled me in with their preference. Okidoke, will update to their preference now.